python/cudf/cudf/core/frame.py

```diff
@@ -84,3 +84,15 @@ def log(self):
 
     def sqrt(self):
         return self._unaryop("sqrt")
+
+    def _repeat(self, count):
```

Comment:
Can we just make this `def repeat` then remove the definition entirely from `DataFrame` and use it via inheritance?

Reply:
WIP. yup.